Tests/PostgresNIOTests/New/Messages/SASLInitialResponseTests.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  File.swift
+//
+//
+//  Created by Fabian Fett on 01.02.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```